src/D2L.Security.OAuth2/Keys/Development/StaticPrivateKeyProvider.cs

```diff
@@ -12,6 +13,10 @@ namespace D2L.Security.OAuth2.Keys.Development {
 	[Obsolete("Only use this in tests and for prototyping.")]
 	internal sealed class StaticPrivateKeyProvider : IPrivateKeyProvider {
 		private readonly Guid m_keyId;
+		[Mutability.Audited(
+			"Todd Lang",
+			"14-Aug-2018",
+			".Net class that is not mutable but can't be marked.")]

```

Comment:
Candidate for KnownImmutableTypes in the analyzer:
https://github.com/Brightspace/D2L.CodeStyle/blob/master/src/D2L.CodeStyle.Analyzers/Immutability/KnownImmutableTypes.cs